src/main/scala/spark/dicom/deidentifier/DicomDeidOptions.scala

```diff
@@ -0,0 +1,69 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+package ai.kaiko.spark.dicom.deidentifier.options
+
+import ai.kaiko.dicom.DicomDeidElem
+
+sealed trait DeidOption {
```

Comment:
`sealed abstract class DeidOption(val priority: Int)`

for shorter code later, e.g.:
`sealed case class RetainUids() extends DeidOption(9) {
`